include/lbann/callbacks/perturb_weights.hpp

```diff
@@ -71,6 +76,18 @@ class perturb_weights : public callback_base {
    *  output_name.
    */
   std::string m_output_name;
+
+   /*
+   *  upper.
+   *  lower.
+   *  scale.
+   *  perturb_probability.
+   */	
+  EvalType m_upper;
+  EvalType m_lower;
+  EvalType m_scale;
+  EvalType m_perturb_probability;

```

Comment:
Let's properly document these variables. If you comment with `///` or `/**`, the comments can be parsed by Doxygen to generate user-friendly documentation.
```suggestion
  /// Upper bound of the perturbed weight
  EvalType m_upper;
  /// Lower bound of the perturbed weight
  EvalType m_lower;
  /// Scale of normal distribution N(1,0) for the perturbed weight
  EvalType m_scale;
  /// The probability of perturbing the weight
  EvalType m_perturb_probability;
```